### DISCUSSION TITLE
Miami 54123, 6Pin, V2B Board - Display flashing on 00H - Help

### DISCUSSION THREAD

**Discussion by sebasnl** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#721] [Q&A]
Hello! I've made my PCB ready this spring and everything worked. Then the weather became really bad so I disconnected my Lay-Z-Spa pump and left it as is. When I Booted it today, the PCB did not come back on in the web, but MQTT did. As I wanted the web surroundings as well, I reinstalled the latest version of the firmware on the ESP8266 module. 

When connected again and changed all settings I got stuck. Neither the web interface, nor the buttons work. Also, after a longer periode (>10 mins) the web interface stops working. These are my configs:
<img width="423" alt="Screenshot 2024-06-07 at 16 46 49" src="https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/assets/61070269/fbcfbdbb-a945-4327-8156-7cdd390112e2">
<img width="423" alt="Screenshot 2024-06-07 at 16 47 02" src="https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/assets/61070269/06100680-094f-4789-b8e3-d9eaf225285c">

Here you can see what the display says:
<img width="435" alt="Screenshot 2024-06-07 at 16 48 18" src="https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/assets/61070269/2c2ed8cd-18a3-4bc1-985b-5fc6cbf70050">

And this is the hwtest information:
<img width="1624" alt="Screenshot 2024-06-07 at 15 57 40" src="https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/assets/61070269/4500b3a9-7e1c-4bc7-82fc-d7b7b69a38f3">

My display only shows '20' and then '00H' with the timer button highlighted. When I touch buttons, every button has a different sound.

My hardware:
Lay-Z-Spa, pre 2021 Miami, Eggshaped pump 54123, 6-pins (!)
![IMG_5176](https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/assets/61070269/4346c60b-d5f2-435b-9cc8-3605528e9265)

ESP8266, PCB V2B, with TXS0108E LLC. 

It used to work under Firmware: 2023-11-19-1330, but now it does not on this firmware, nor on the newest. 

https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/assets/61070269/f98b81ca-766d-4480-878c-4f15ea98a74b

Could you help me further? What am I doing wrong?

**Comment by visualapproach** [comment]
What you do right is posting a detailed report. Very nice!

So, since it worked before, something should've changed right? Moisture on the pcb? Corrosion? Or just wrong model chosen?
Does it work without the ESP?

**Reply by sebasnl** [reply]
Thank you so much for your response! I don't know what went wrong; I have 4 ESP's at home and I flashed them all and then installed them all with the latest firmware. All of a sudden today it seems to work. Coffee is on its way, I am super happy with this project :-)

Ps. the PCB is in a zip-lock bag with a small silica bag to eliminate water/fog/condensation problems. It's bone dry and the PCB and other elements look great.

**Reply by visualapproach** [reply]
Thanks man, much appreciated 😊
Hope it works a long time now!
